infra/experimental/SystemSan/SystemSan.cpp

```diff
@@ -302,7 +302,8 @@ void inspect_for_arbitrary_file_open(pid_t pid, const user_regs_struct &regs) {
     log_file_open(path, regs.rdx);
     return;
   }
-  if (path[0] == '/' && path.length() > 1) {
+  // A file name at root whose length is less than 2
+  if (path[0] == '/' && path.length() > 1 && path.length() <= 3) {
```

Comment:
This is a bit hacky. Also the comment doesn't match. 

How about matching on non-printable characters instead?

Reply:
Comment matches : `path.length() <= 3` means one char for `/`+ A file name at root whose length is less than 2 ;-)

> How about matching on non-printable characters instead?

Sounds good :-)

Rebased and force-pushed